hack/README.md

```diff
@@ -131,12 +131,12 @@ Environment Variables and Supported Values
 
 | Env Variable       | Default       | Additional Values           |
 |--------------------|---------------|-----------------------------|
-|KUBEVIRT_PROVIDER   | k8s-1.16      | k8s-1.17, os-3.11.0-crio,   |
+|KUBEVIRT_PROVIDER   | k8s-1.18      | k8s-1.17, os-3.11.0-crio,   |
 |KUBEVIRT_STORAGE*   | none          | ceph, hpp, nfs, ember_lvm   |
 |KUBEVIRT_PROVIDER_EXTRA_ARGS |      |                             |
 |NUM_NODES           | 1             | 2-5                         |
 
-KUBEVIRT_STORAGE is only supported for k8s-1.16 and OKD 4.3 providers
+KUBEVIRT_STORAGE is only supported for k8s-1.18 and OKD 4.3 providers
```

Comment:
This is not true anymore, the 1.17 provider supports the different storages as well. Also the OKD 4.3 provider no longer works, so we should take that part of the message out.